cmd/manager/auto_exit_noprocess.go

```diff
@@ -25,7 +25,20 @@ var (
 )
 
 func fetchProcesses() (processes, error) {
-	return process.AllProcesses()
+	ps, err := process.Processes()
+	if err != nil {
+		return nil, err
+	}
+
+	procs := make(processes)
+	for _, p := range ps {
+		name, err := p.Name()
+		if err != nil {
+			return nil, err
+		}
```

Comment:
I think we might want to ignore the error (ie. just debug log it), if a process name cannot be fetched the most likely cause is that the process was stopped.
If returning an error is what `process.AllProcesses` did then feel free to ignore my comment

Reply:
Updated to log